lib/CustomFields/components/FieldAccordion/FieldAccordion.js

```diff
@@ -15,12 +15,14 @@ const editFieldsByType = {
   [fieldTypes.TEXTBOX_SHORT]: editFields.TextboxFields,
   [fieldTypes.TEXTBOX_LONG]: editFields.TextboxFields,
   [fieldTypes.SINGLE_CHECKBOX]: editFields.CheckboxFields,
+  [fieldTypes.SINGLE_SELECT_DROPDOWN]: editFields.SingleSelectDropdown,
 };
 
 const viewSectionsByType = {
   [fieldTypes.TEXTBOX_SHORT]: viewSections.TextboxViewSection,
   [fieldTypes.TEXTBOX_LONG]: viewSections.TextboxViewSection,
   [fieldTypes.SINGLE_CHECKBOX]: viewSections.CheckboxViewSection,
+  [fieldTypes.SINGLE_SELECT_DROPDOWN]: viewSections.SingleSelectDropdownSection,
 };
```

Comment:
This is a bit of a tangent, but: instead of inventing new names for these element types (`TEXTBOX_SHORT`, `TEXTBOX_LONG`, ...) can we use the matching names for the components from stripes-components these types will be rendered as, e.g. `TEXTFIELD`, `TEXTAREA`, `CHECKBOX`, `SELECT`? 

For other folks working with this code, I think it would be valuable to have consistency in how the same things are named across multiple repositories.